scripts/start.sh

```diff
@@ -377,6 +377,17 @@ else
   echo ${ARLAS_HUB_APP_PATH}  "is used as app base path "
 fi
 
+# Set Tab title name
+if [ -z "${ARLAS_TAB_NAME}" ]; then
+  ARLAS_TAB_NAME=""
```

Comment:
I think we should put `ARLAS Hub` as a default value, otherwise we will have a black title on the tab no ?

Reply:
ok